roles/openshift_etcd_facts/vars/main.yml

```diff
@@ -5,6 +5,6 @@ etcd_hostname: "{{ openshift.common.hostname }}"
 etcd_ip: "{{ openshift.common.ip }}"
 etcd_cert_subdir: "etcd-{{ openshift.common.hostname }}"
 etcd_cert_prefix:
-etcd_cert_config_dir: "{{ '/etc/etcd' if not openshift.common.is_etcd_system_container | bool else '/var/lib/etcd/etcd.etcd/etc' }}"
+etcd_cert_config_dir: "/etc/etcd"
```

Comment:
Think it would make sense to add `etcd_system_container_cert_config_dir: /var/lib/etcd/etcd.etcd/etc` var here as well and use it instead of `r_etcd_common_system_container_host_dir` where the certs are handled?